js/adapt-contrib-bookmarking.js

```diff
@@ -34,10 +54,37 @@ class Bookmarking extends Backbone.Controller {
       'menuView:ready': this.setupMenu,
       'pageView:preRender': this.setupPage,
       'view:childAdded': this.onChildViewAdded,
-      'view:preRemove': this.onChildViewPreRemove
+      'view:preRemove': this.onChildViewPreRemove,
+      'bookmarking:resume': this.navigateToFurthest
```

Comment:
This event handler is not correct.
```suggestion
      'bookmarking:resume': this.navigateTo
```